data/events/2024-belo-horizonte.yml

```
@@ -0,0 +1,131 @@
+
+
+# These are the same people you have on the mailing list and Slack channel.
+team_members: # Name is the only required field for team members.
+  - name: "Paloma Lataliza"
+    employer: "Cloud Engineer at Claro Brasil"
+    twitter: "shescloud_"
+    bio: "Apaixonada por tecnologia, entusiasta em cloud, fã de cavaleiros do zodíaco e que ama compartilhar conhecimento. Graduada em Ciência da Computação pela Universidade FUMEC e pós graduanda em Cloud Computing pela PUC Minas. Sou uma AWS Community Builder, umas das Líderes do AWS User Group MG e Co-Fundadora do Mulheres na Nuvem MG."
+    image: "paloma.jpg"
+
+  - name: "Rafaela Vidotti"
+    employer: "Project Manager at Bid Broke"
+    linkedin: "https://www.linkedin.com/in/rafaelavidotti/"
+    bio: "TikToker de Dancinhas e Infraestutura de TI que ama fugir de um padrão, luta diariamente por mais mulheres no ambiente, e para desmistificar “essa tal de TI “ para todos. Recém-palestrante, líder de comunidade AWSUG e fundadora de uma para mulheres."
+    image: "rafaela.jpeg"
+
+  - name: "Victor Parreira"
+    employer: "Infrastructure Analyst at Quality Digital"
+    linkedin: "https://www.linkedin.com/in/victorparreira/"
+    bio: "Um cara de humanas que descobriu em tech sua maior paixão. Gosto de misturar tudo aquilo que aprendi na minha carreira até agora para trazer uma nova visão sobre o mundo de cloud e suas ferramentas.  <br> <br> <br> <br> <br>"
+    image: "victor.jpg" 
+
+  - name: "Arthur Gomes"
+    employer: "Consultor DevOps na Arkhi Business Agility"
+    linkedin: "https://www.linkedin.com/in/arthur-faria-97b676144/"
+    bio: "Consultor sênior de DevOps em horário comercial, Barista amador em horário de lazer, e eterno estudante de Engenharia de Sistemas na UFMG nos intervalos! Com uma grande paixão por otimizar o tempo e uma predileção pelo universo DevOps (tanto a cultura quanto as ferramentas), eu vivo pelo lema: 'Se é preciso fazer mais de uma vez, é melhor automatizar'"
+    image: "arthur.jpeg"  
+  
+  - name: "Zandler Oliveira"
+    twitter: "zandler"
+    linkedin: "https://www.linkedin.com/in/zandler-oliveira/"
+    bio: ""
+    image: "zandler-oliveira.jpg"
+
+  - name: "Bernardo Machado"
+    instagram: "bmachado_devops"
+    linkedin: "https://www.linkedin.com/in/bernardo-machado-68096960"
+    bio: "Devsecops"
+    image: "bernardo.jpeg"
+
+  - name: "Wellington Brandão"
+    instagram: "wnbrandao"
+    linkedin: " http://linkedin.com/in/wnbrandao"
+    bio: "Devops"
+    image: "wellington.jpeg"
+
+  - name: "Daniel Rodrigues"
+    employer: "Tech Lead at Flash"
+    linkedin: "https://www.linkedin.com/in/eusouodaniel"
+    bio: "Amplo conhecimento e experiência profissional em desenvolvimento web, com conhecimento em diversas ferramentas e linguagens, facilidade no aprendizado e alto foco. Atualmente atua como Engenheiro de Software, Mentor e Professor."
+    image: "daniel-rodrigues.jpeg"
+
+organizer_email: "" # Put your organizer email address here
```

This needs to be populated as it is used in templates elsewhere (the sponsor page, for example).

```suggestion
organizer_email: "belo-horizonte@devopsdays.org" # Put your organizer email address here
```